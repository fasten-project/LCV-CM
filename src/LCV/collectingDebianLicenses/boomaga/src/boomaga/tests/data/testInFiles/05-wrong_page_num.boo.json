{
    "checksum": "ff2ebdfb79c0ac5c0c521fd88d3975f5ced765408632bf558d7e5fe14fcb10b8",
    "file": "05-wrong_page_num.boo",
    "mime": {
        "encoding": "binary",
        "type": "application/octet-stream"
    },
    "number_of_duplicates": 1,
    "package": "boomaga",
    "path": "boomaga/3.0.0-1/src/boomaga/tests/data/testInFiles/05-wrong_page_num.boo",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/boomaga/3.0.0-1/",
        "metric": {
            "size": 5604
        },
        "pts_link": "https://tracker.debian.org/pkg/boomaga",
        "sloc": [
            [
                "cpp",
                13687
            ],
            [
                "sh",
                314
            ],
            [
                "xml",
                9
            ],
            [
                "makefile",
                8
            ]
        ],
        "suites": [
            "bookworm",
            "bullseye",
            "sid"
        ],
        "vcs_browser": "https://gitlab.com/perezmeyer/debian-package-boomaga",
        "vcs_type": "git"
    },
    "raw_url": "/data/main/b/boomaga/3.0.0-1/src/boomaga/tests/data/testInFiles/05-wrong_page_num.boo",
    "stat": {
        "perms": "rw-r--r--",
        "size": 8755,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": false,
    "type": "file",
    "version": "3.0.0-1"
}