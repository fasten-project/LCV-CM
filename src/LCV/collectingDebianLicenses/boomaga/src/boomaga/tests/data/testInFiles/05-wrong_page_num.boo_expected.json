{
    "checksum": "9feeb4caa2e2a155ad2aef60e0f9c333be27e90420f99f5dff8a1e8383312756",
    "file": "05-wrong_page_num.boo_expected",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/plain"
    },
    "number_of_duplicates": 2,
    "package": "boomaga",
    "path": "boomaga/3.0.0-1/src/boomaga/tests/data/testInFiles/05-wrong_page_num.boo_expected",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/boomaga/3.0.0-1/",
        "metric": {
            "size": 5604
        },
        "pts_link": "https://tracker.debian.org/pkg/boomaga",
        "sloc": [
            [
                "cpp",
                13687
            ],
            [
                "sh",
                314
            ],
            [
                "xml",
                9
            ],
            [
                "makefile",
                8
            ]
        ],
        "suites": [
            "bookworm",
            "bullseye",
            "sid"
        ],
        "vcs_browser": "https://gitlab.com/perezmeyer/debian-package-boomaga",
        "vcs_type": "git"
    },
    "raw_url": "/data/main/b/boomaga/3.0.0-1/src/boomaga/tests/data/testInFiles/05-wrong_page_num.boo_expected",
    "stat": {
        "perms": "rw-r--r--",
        "size": 429,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "3.0.0-1"
}